lib/merge.js

```
@@ -255,6 +274,26 @@ function tryMerge(
   );
 }
 
+function getMergeMethod(defaultMergeMethod, mergeMethodLabels, pullRequest) {
+  const foundMergeMethodLabels = pullRequest.labels.flatMap(l =>
+    mergeMethodLabels.filter(ml => ml.label === l.name)
+  );
+  if (foundMergeMethodLabels.length > 0) {
+    const first = foundMergeMethodLabels[0];
+    if (foundMergeMethodLabels.length > 1) {
+      logger.info(
+        `Discovered multiple merge method labels, will merge with method ${first.method}`
+      );
```

This should throw an error instead, IMO